README.md

```diff
@@ -154,6 +155,75 @@ export default function () {
 }
 ```
 
+## Helpers
+
+The `xk6-kubernetes` extension offers helpers to facilitate common tasks when setting up a tests. All helper functions work in a implicit namespace to facilitate the development of tests segregated by namespace. The helpers are accessed using the following methods:
+
+|  Method      | Parameters|   Description |
+| -------------| ---| ------ |
+| helpers      | | returns helpers that operate in the default namespace |
+| namespacedHelpers | namespace | returns helpers that operate in the given namespace |
```

Comment:
I'm not sure how I feel about this.

Thoughts on having only one method? Where namespace is passed as a parameter, and if it's not, it defaults to default.

Reply:
I'm against passing empty parameters in go for signaling a null value as it is difficult to catch errors. I prefer to make explicit the choice of.

Reply:
Then, yeah, I would go with that, forcing namespace passing 👍